river/tree/splitter/__init__.py

```diff
@@ -0,0 +1,21 @@
+from .base_splitter import Splitter
+from .ebst_splitter import EBSTSplitter
+from .exhaustive_splitter import ExhaustiveSplitter
```

Comment:
This also could be named `BSTSplitter`. Some papers do that if I recall it correctly. The regression version "extends" this algorithm, so it is called "Extended Binary Search Tree {Splitter} Attribute Observer" -> `EBSTSplitter`.

Comments, @jacobmontiel?

Reply:
Let's keep the name ExhaustiveSplitter since this is the common choice in the literature.